docs/data/tree-view/simple-tree-view/customization/customization.md

```diff
@@ -9,24 +9,61 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
 
 # Simple Tree View - Customization
 
-<p class="description">Customize the rendering of your Tree View.</p>
+<p class="description">Learn how to customize the Tree View component.</p>
 
-## Custom icons, border and animation
+## Basics
 
-{{"demo": "CustomizedTreeView.js"}}
+### Custom icons
+
+Use the `defaultCollapseIcon`, `defaultExpandIcon`, and `defaultEndIcon` props to customize the Tree View icons.
+You can use either icon libraries, such as [Material Icons](/material-ui/material-icons/), or create your own.
+
+{{"demo": "CustomIcons.js", "defaultCodeOpen": false}}
+
+### Custom toggle animations
+
+Use the `TransitionComponent` prop on the `TreeItem` to pass a component that handles your animation.
+
+The below demo animates Material UI's [Collapse](/material-ui/transitions/#collapse) with [react-spring](https://www.react-spring.dev/).
+
+{{"demo": "CustomAnimation.js", "defaultCodeOpen": false}}
+
+### Custom styling
+
+Use the `treeItemClasses` to target internal elements of the Tree Item component and change their styles.
```

Comment:
```suggestion
Use `treeItemClasses` to target internal elements of the Tree Item component and change their styles.
```

Or we could alternatively say what `treeItemClasses` is, which I'm not sure whether it's a hook, class, etc.

```suggestion
Use the `treeItemClasses` {hook?}{class?} to target internal elements of the Tree Item component and change their styles.
```

Reply:
Technically it's an object that gives you the css classes that the `TreeItem` has, but I'm not sure it's worth specifying that, so I think we should go with the first suggestion 👍